coordinator/src/services/translation/translation-service.ts

```diff
@@ -87,6 +83,19 @@ function createSignedInfo(digestValue: string): XmlJs.ElementCompact {
   } as XmlJs.ElementCompact
 }
 
+function createBase64Display(): string {
+  const hardCodedResponse = "####Patient**NHS Number**: 945 374 0586**Name**: PENSON, HEADLEY TED (Mr)**Date of Birth**: 1977-03-27**Address (Home)**:  10 CRECY CLOSE,  DERBY,  DE22 3JU####Author**Name**: CHANDLER, ANDREW**Telecom (Work)**: 01945700223####Organisation**Name**: PARSON DROVE SURGERY**Telecom (Work)**: 01945700223**Address (Work)**:  240 MAIN ROAD,  PARSON DROVE,  WISBECH,  CAMBRIDGESHIRE,  PE13 4JA####Medication Requested|Name|Dose|Quantity|Unit||----|----|--------|----||Microgynon 30 tablets (Bayer Plc)|As Directed|63|tabletetc."
```

Comment:
I think the line breaks have gone missing here

Reply:
Fixed